opp_gui/src/widgets/tool_path_parameters_editor_widget.cpp

```diff
@@ -82,15 +86,38 @@ noether_msgs::ToolPathConfig ToolPathParametersEditorWidget::getToolPathConfig()
   noether_msgs::ToolPathConfig config;
 
   // Create a path configuration from the line edit fields
+  config.type = noether_msgs::ToolPathConfig::PLANE_SLICER_RASTER_GENERATOR;  // was SURFACE_WALK_RASTER_GENERATOR;
+
   config.surface_walk_generator.point_spacing = ui_->double_spin_box_point_spacing->value();
   config.surface_walk_generator.tool_offset = ui_->double_spin_box_tool_z_offset->value();
   config.surface_walk_generator.raster_spacing = ui_->double_spin_box_line_spacing->value();
   config.surface_walk_generator.min_hole_size = ui_->double_spin_box_min_hole_size->value();
   config.surface_walk_generator.min_segment_size = ui_->double_spin_box_min_segment_length->value();
   config.surface_walk_generator.intersection_plane_height = ui_->double_spin_box_intersecting_plane_height->value();
   config.surface_walk_generator.raster_rot_offset = ui_->double_spin_box_raster_angle->value() * M_PI / 180.0;
-  config.surface_walk_generator.raster_wrt_global_axes = false;
```

Comment:
I'm leaving it in, because someone may not be happy with the change to the Plane slicer and want to go back to the way that it was. There are some parameters in the surface walk, that are part of the gui that are not used by the plane slicer. I know its ugly code. This whole gui needs to be designed rather than grown.